src/wp-includes/rest-api/endpoints/class-wp-rest-settings-controller.php

```diff
@@ -65,7 +67,20 @@ public function register_routes() {
 	 * @return bool True if the request has read access for the item, otherwise false.
```

Comment:
Minor: I think this docblock should get an update to indicate that in 6.8 it changed the behavior.